src/components/ebay-listbox-button/index.marko

```diff
@@ -13,62 +12,90 @@ $ const {
     truncate,
     prefixLabel,
     prefixId,
-    unselectedText = '-',
+    unselectedText = "-",
     floatingLabel,
     collapseOnSelect,
     listSelection,
     variant,
     hasError,
+    split = "none",
+    a11yIconPrefixText,
     ariaDescribedby: describedby,
     ...htmlInput
 } = input;
-
 $ var selectedOption = (options as Option[])[state.selectedIndex];
 $ var selectedText = selectedOption && selectedOption.text;
+$ var selectedIcon = selectedOption && selectedOption.icon;
+$ var a11ySelectedIconText = selectedIcon && selectedOption && selectedOption.text;
 $ var labelId = prefixId && component.getElId("label");
 $ var displayText = selectedText || unselectedText;
-$ var isForm = variant === 'form';
+$ var isForm = variant === "form";
+
+<macro|{ alwaysDisplay }: { alwaysDisplay: boolean }| name="render-text">
+    <if(selectedIcon)>
+        <span id=labelId class="btn__text">
+            <${selectedIcon}/>
+        </span>
+    </if>
+    <else-if(selectedText || alwaysDisplay)>
+        <span id=labelId class="btn__text">
+            ${displayText}
+        </span>
+    </else-if>
+</macro>
 
 <${truncate && !fluid ? "div" : "span"}
     ...processHtmlAttributes(htmlInput)
     key="container"
-    class=["listbox-button", inputClass, fluid && "listbox-button--fluid", isForm && `listbox-button--form`, hasError && 'listbox-button--error']
+    class=[
+        "listbox-button",
+        inputClass,
+        fluid && "listbox-button--fluid",
+        isForm && `listbox-button--form`,
+        hasError && "listbox-button--error",
+    ]
     on-expander-expand("handleExpand")
-    on-expander-collapse("handleCollapse")>
+    on-expander-collapse("handleCollapse")
+>
     <button
         key="button"
         class=[
             "listbox-button__control",
-            'btn',
-            borderless ? "btn--borderless" : 'btn--form',
+            "btn",
+            split !== "none" && `btn--split-${split}`,
+            borderless ? "btn--borderless" : "btn--form",
             truncate && "btn--truncated",
-            floatingLabel && "btn--floating-label"
+            floatingLabel && "btn--floating-label",
         ]
+        aria-label=a11ySelectedIconText && `${a11yIconPrefixText}: ${a11ySelectedIconText}`
         value=selectedText
         type="button"
         disabled=disabled
```

Comment:
Disabled is not disabling the look/feel of the flags. 
<img width="1714" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/06811eba-639e-47f6-8132-1f96c8e0d3ec">

Reply:
I don't see it like this
Did you link skin? Im using `17.4.0`

<img width="341" alt="Screenshot 2024-04-18 at 8 00 17 AM" src="https://github.com/eBay/ebayui-core/assets/1755269/e4af58c7-f5db-416d-909e-71df5006ccc1">
<img width="330" alt="Screenshot 2024-04-18 at 8 00 57 AM" src="https://github.com/eBay/ebayui-core/assets/1755269/6909cef9-3e6f-4e30-8b1a-245daaf7d766">